pkgs/development/python-modules/gensim.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, python
+, numpy
+, scipy
+, six
+, smart_open
+, testfixtures
+, unittest2
+, pyro4
+, pyemd
+, scikitlearn
+, morfessor
+}:
+buildPythonPackage rec {
+  name = "${pname}-${version}";
+  pname = "gensim";
+  version = "2.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1wn7bji9b80wn1yggmh7a0dlwzdjr6cp24x4p33j2rf29lxnm2kc";
+  };
+
+  propagatedBuildInputs = [
+   numpy
+   scipy
+   six
+   smart_open
+  ];
+  
+  buildInputs = [
+    testfixtures
+    unittest2
+    pyro4
+    pyemd
+    scikitlearn
+    morfessor
+  ];
+
+  checkPhase = ''
```

Comment:
Note that currently this is the default `checkPhase` so its not necessary. No need to change it though.